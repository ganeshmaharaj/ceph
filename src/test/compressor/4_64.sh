#!/bin/sh

#"8K/4/64"
CEPH_ARGS="--bluestore_max_ops 4 --async_compressor_threads 64 --bluestore_compression_min_blob_size 8192 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

#"16K/4/64"
CEPH_ARGS="--bluestore_max_ops 4 --async_compressor_threads 64 --bluestore_compression_min_blob_size 16384 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

#"64K/4/64"
CEPH_ARGS="--bluestore_max_ops 4 --async_compressor_threads 64 --bluestore_compression_min_blob_size 65536 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

#"256K/4/64"
CEPH_ARGS="--bluestore_max_ops 4 --async_compressor_threads 64 --bluestore_compression_min_blob_size 262144 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

#"4M/4/64"
CEPH_ARGS="--bluestore_max_ops 4 --async_compressor_threads 64 --bluestore_compression_min_blob_size 4194304 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

