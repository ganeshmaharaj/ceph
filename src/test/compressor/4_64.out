Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/8192 in 1342 msec. cores/threads = 4/64
ctx switches 1601, use(%) 94, user(%) 93, kernel(%) 0
[       OK ] PerfTest.perf_compare_dummy (5322 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 10287mb ratio=0.449576 via snappy/8192 in 14959 msec. cores/threads = 4/64
ctx switches 15597, use(%) 98, user(%) 98, kernel(%) 0
[       OK ] PerfTest.perf_compare_snappy (18508 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 9458mb ratio=0.41334 via zlib+isal/8192 in 24337 msec. cores/threads = 4/64
ctx switches 25720, use(%) 99, user(%) 98, kernel(%) 0
[       OK ] PerfTest.perf_compare_isal (27887 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 7512mb ratio=0.328286 via zlib/8192 in 92474 msec. cores/threads = 4/64
ctx switches 100481, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_zlib (96015 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 8086mb ratio=0.35336 via zlib+qat/8192 in 26776 msec. cores/threads = 4/64
ctx switches 4463102, use(%) 105, user(%) 42, kernel(%) 62
[       OK ] PerfTest.perf_compare_qat (33005 ms)
[----------] 5 tests from PerfTest (180738 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (180738 ms total)
[  PASSED  ] 5 tests.
Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/16384 in 1169 msec. cores/threads = 4/64
ctx switches 1418, use(%) 97, user(%) 96, kernel(%) 1
[       OK ] PerfTest.perf_compare_dummy (4895 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 9731mb ratio=0.425288 via snappy/16384 in 14519 msec. cores/threads = 4/64
ctx switches 15722, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_snappy (17975 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 8862mb ratio=0.387299 via zlib+isal/16384 in 20577 msec. cores/threads = 4/64
ctx switches 21758, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_isal (24084 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 7102mb ratio=0.3104 via zlib/16384 in 77335 msec. cores/threads = 4/64
ctx switches 83747, use(%) 100, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_zlib (80751 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 7774mb ratio=0.339746 via zlib+qat/16384 in 14626 msec. cores/threads = 4/64
ctx switches 2225424, use(%) 102, user(%) 46, kernel(%) 55
[       OK ] PerfTest.perf_compare_qat (21004 ms)
[----------] 5 tests from PerfTest (148710 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (148710 ms total)
[  PASSED  ] 5 tests.
Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/65536 in 1013 msec. cores/threads = 4/64
ctx switches 1258, use(%) 99, user(%) 98, kernel(%) 0
[       OK ] PerfTest.perf_compare_dummy (4478 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 9044mb ratio=0.395235 via snappy/65536 in 16011 msec. cores/threads = 4/64
ctx switches 20521, use(%) 99, user(%) 90, kernel(%) 9
[       OK ] PerfTest.perf_compare_snappy (19267 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 8421mb ratio=0.36801 via zlib+isal/65536 in 18799 msec. cores/threads = 4/64
ctx switches 21430, use(%) 99, user(%) 97, kernel(%) 1
[       OK ] PerfTest.perf_compare_isal (22099 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 6675mb ratio=0.29174 via zlib/65536 in 81087 msec. cores/threads = 4/64
ctx switches 89017, use(%) 99, user(%) 93, kernel(%) 6
[       OK ] PerfTest.perf_compare_zlib (84370 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 7484mb ratio=0.327073 via zlib+qat/65536 in 7503 msec. cores/threads = 4/64
ctx switches 660945, use(%) 90, user(%) 58, kernel(%) 31
[       OK ] PerfTest.perf_compare_qat (13003 ms)
[----------] 5 tests from PerfTest (143218 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (143218 ms total)
[  PASSED  ] 5 tests.
Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/262144 in 1237 msec. cores/threads = 4/64
ctx switches 3781, use(%) 96, user(%) 94, kernel(%) 1
[       OK ] PerfTest.perf_compare_dummy (4603 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 9043mb ratio=0.395201 via snappy/262144 in 45524 msec. cores/threads = 4/64
ctx switches 3041387, use(%) 62, user(%) 51, kernel(%) 11
[       OK ] PerfTest.perf_compare_snappy (48787 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 8310mb ratio=0.363192 via zlib+isal/262144 in 17671 msec. cores/threads = 4/64
ctx switches 20102, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_isal (21022 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 6564mb ratio=0.286855 via zlib/262144 in 81903 msec. cores/threads = 4/64
ctx switches 93524, use(%) 99, user(%) 96, kernel(%) 3
[       OK ] PerfTest.perf_compare_zlib (85112 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 7420mb ratio=0.324286 via zlib+qat/262144 in 9934 msec. cores/threads = 4/64
ctx switches 653354, use(%) 56, user(%) 48, kernel(%) 8
[       OK ] PerfTest.perf_compare_qat (15002 ms)
[----------] 5 tests from PerfTest (174526 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (174526 ms total)
[  PASSED  ] 5 tests.
Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/4194304 in 2969 msec. cores/threads = 4/64
ctx switches 5075, use(%) 97, user(%) 96, kernel(%) 1
[       OK ] PerfTest.perf_compare_dummy (6349 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 9043mb ratio=0.39519 via snappy/4194304 in 47750 msec. cores/threads = 4/64
ctx switches 202877, use(%) 54, user(%) 52, kernel(%) 2
[       OK ] PerfTest.perf_compare_snappy (50840 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 8276mb ratio=0.361688 via zlib+isal/4194304 in 18763 msec. cores/threads = 4/64
ctx switches 22055, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_isal (21902 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 6528mb ratio=0.285316 via zlib/4194304 in 79901 msec. cores/threads = 4/64
ctx switches 86546, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_zlib (82988 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 7399mb ratio=0.323362 via zlib+qat/4194304 in 10260 msec. cores/threads = 4/64
ctx switches 573036, use(%) 55, user(%) 50, kernel(%) 5
[       OK ] PerfTest.perf_compare_qat (15003 ms)
[----------] 5 tests from PerfTest (177082 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (177082 ms total)
[  PASSED  ] 5 tests.
